make.go

```diff
@@ -468,7 +468,7 @@ func createGitRepository(debsrc, gopkg, orig string, u *upstream,
 
 	// Preconfigure branches
 
-	branches := []string{debianBranch, "upstream"}
+	branches := []string{debianBranch, "upstream/latest"}
```

Comment:
Should this behavior only change when dep14 is enabled?

Reply:
This is inside `createGitRepository()` and defines the default, which we are now intentionally changing. I think this is correct but of course I might be wrong?

The `gbp.conf: dep14=True` is something that is passed to [pbuilder](https://manpages.debian.org/unstable/pbuilder/pbuilder.8.en.html).

Reply:
Actually, I see the shortcoming now, thanks for pointing it out. Refactored this in latest commit to define a default for `upstreamImportBranch` and then override it in a dep14 check.